.github/actions/bumpVersion/bumpVersion.js

```diff
@@ -47,26 +55,40 @@ function generateAndroidVersionCode(npmVersion) {
 }
 
 /**
- * Use the react-native-version package to update the version of a native platform.
+ * Update the Android app version.
  *
- * @param {String} platform – "ios" or "android"
- * @param {String} versionCode – The version code to update the native platform.
+ * @param {String} versionName
+ * @param {String} versionCode
  */
-function updateNativeVersion(platform, versionCode) {
-    if (platform !== 'android' && platform !== 'ios') {
-        console.error('Invalid native platform specified!', platform);
-        core.setFailed();
-        return;
-    }
+function updateAndroidVersion(versionName, versionCode) {
+    console.log('Updating android:', `versionName: ${versionName}`, `versionCode: ${versionCode}`);
+    readFileAsync(BUILD_GRADLE_PATH, {encoding: 'utf8'})
+        .then((content) => {
+            let updatedContent = content.replace(/versionName "([0-9.-]*)"/, `versionName "${versionName}"`);
+            updatedContent = updatedContent.replace(/versionCode ([0-9]*)/, `versionCode ${versionCode}`);
+            fs.writeFile(BUILD_GRADLE_PATH, updatedContent, () => {});
+        });
+}
 
-    // Note: We're using `exec` instead of the `react-native-version` javascript sdk to avoid a known issue
-    // with `ncc` and `process.cwd` https://github.com/vercel/webpack-asset-relocator-loader/issues/112
-    exec(`react-native-version --amend --target ${platform} --set-build ${versionCode}`)
+/**
+ * Update the iOS app version.
+ *
+ * @param {String} version
+ */
+function updateiOSVersion(version) {
+    const shortVersion = version.slice(0, -4);
```

Comment:
Doesn't this assume build will always be 3 digits?

Reply:
Maybe?

```suggestion
    const shortVersion = version.split("-")[0];
```

Reply:
updated!